mypy/nodes.py

```diff
@@ -270,6 +310,20 @@ def name(self) -> str:
     def accept(self, visitor: NodeVisitor[T]) -> T:
         return visitor.visit_overloaded_func_def(self)
 
+    def serialize(self) -> JsonDict:
+        return {'.class': 'OverloadedFuncDef',
```

Comment:
I wonder if `is_property` and `_fullname` should be preserved (inherited from `FuncBase`). Also, does `info` get set somewhere?